.github/workflows/ci-test.yml

```diff
@@ -92,7 +92,10 @@ jobs:
     container: python:3.10-slim-buster
     env:
       ANSIBLE_COLLECTIONS_PATH: /work-dir
-      SKIP_TESTS: "vm_import|vm_export|dns_config|time_server|time_zone"
+      # vm_import|vm_export - they need SMB server
+      # dns_config|time_server - NTP cannot be reconfigured if DNS is invalid
+      # git_issues - slow, do not run on each push. TODO - run them only once a day
```

Comment:
**GIt_issues**
How about pull_request only?
Either when pull request is created or when it's approved.
Should be run at least before merge to check if any existing code is broken.

Reply:
Dispatch was the agreed upon solution.
Github_issues can be manually triggered before the code is merged into main to prevent any existing code breaking.
👍

Reply:
will be run on push/merge to main, and manually (workflow-dispatch)